jobs/rocoto/marineanlchkpt.sh

```diff
@@ -8,11 +8,10 @@ source "${HOMEgfs}/ush/preamble.sh"
 status=$?
 [[ ${status} -ne 0 ]] && exit "${status}"
 
-export job="ocnanalrun"
-export jobid="${job}.$$"
+export job="marineanlchkpt"
```

Comment:
why remove `jobid`?

Reply:
no reason, will fix.

Reply:
init, variational, checkpoint and finalize all run in the same "marinevariational" scratch dir, which has no job id. Is this an issue?